frontend/src/pages/ErgebnisPage.js

```diff
@@ -0,0 +1,39 @@
+import Page from "../components/Page";
+import {Link} from "react-router-dom";
+import React from "react";
+import styled from "styled-components/macro";
+import HeaderErgebnis from "../components/HeaderErgebnis";
+
+export default function ErgebnisPage() {
+    return (
+        <Page>
+            <HeaderErgebnis/>
+            <Wrapper>
+                <CardContainer>
+                    <p>
+                        Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt
+                        ut labore et dolore magna aliquyam erat, sed diam voluptua.
+                        At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea
+                        takimata sanctus est Lorem ipsum dolor sit amet.
+                    </p>
+                </CardContainer>
+            </Wrapper>
+            <button><Link to={'/manual'}>Start Test</Link></button>
+        </Page>
+    )
+}
+const CardContainer = styled.section`
+  display: block;;
```

Comment:
```suggestion
  display: block;
```
gleiche wie oben, doppelte ;;